docs/usage.md

```diff
@@ -113,15 +111,15 @@ This version number will be logged in reports when you run the pipeline, so that
 
 ## Main arguments
 
-### `--dia_mzmls`
+### `--input`
 
 Use this to specify the location of your input dia raw files (mzML). For example:
```

Comment:
All the main arguments documentation can now be removed, as it is part of the parameter json